app/src/main/java/org/simple/clinic/registration/phone/loggedout/LoggedOutOnThisDeviceDialog.kt

```diff
@@ -0,0 +1,104 @@
+package org.simple.clinic.registration.phone.loggedout
+
+import android.annotation.SuppressLint
+import android.app.Dialog
+import android.content.DialogInterface
+import android.os.Bundle
+import android.widget.Button
+import androidx.appcompat.app.AlertDialog
+import androidx.appcompat.app.AppCompatDialogFragment
+import androidx.fragment.app.FragmentManager
+import io.reactivex.Observable
+import io.reactivex.subjects.PublishSubject
+import org.simple.clinic.R
+import org.simple.clinic.activity.TheActivity
+import org.simple.clinic.bindUiToControllerWithoutDelay
+import org.simple.clinic.util.unsafeLazy
+import org.simple.clinic.widgets.ScreenCreated
+import org.simple.clinic.widgets.ScreenDestroyed
+import org.simple.clinic.widgets.UiEvent
+import javax.inject.Inject
+
+class LoggedOutOnThisDeviceDialog : AppCompatDialogFragment() {
```

Comment:
Yeah, it can be. We already have a `LoggedOutOnOtherDeviceDialog` which is shown when the user enters the OTP and logs in, so this for me seemed like a good name for its counterpart. I can rename it if you feel this sounds better, though.

Reply:
Done.